bookwyrm/templates/get_started/book_preview.html

```diff
@@ -1,8 +1,8 @@
 {% load i18n %}
-<div class="column is-narrow is-clipped has-text-centered">
-    {% include 'snippets/book_cover.html' with book=book %}
-    <label class="label" for="id_shelve_{{ book.id }}">
```

Comment:
I could not see any use for this unclosed `<label>`, let me know if I missed something.

Reply:
you didn't miss anything, that shouldn't be there